addons/mail/static/src/core/public_web/discuss.js

```diff
@@ -67,6 +69,11 @@ export class Discuss extends Component {
             },
             { capture: true }
         );
+        onRendered(() => {
```

Comment:
I'm not sure but shouldn't it be a `useEffect` instead?

If `displayName` is not used during rendering, this would fail to update it on renaming thread.

Reply:
`useEffect` might not trigger either in that case actually, but at least it will avoid resetting title when it doesn't change.